client-src/default/polyfill.js

```diff
@@ -0,0 +1,4 @@
+'use strict';
+
+require('core-js/features/string/includes');
+require('core-js/features/array/includes');
```

Comment:
We pollution global namespace :disappointed: We should use name import like in docs https://github.com/zloirock/core-js

Reply:
sorry, why?

Reply:
Maybe we need setup support `import`/`export` in our babel config for client too

Reply:
Using `require('core-js/features/array/includes');` modify `Array.prototype`, so if developer already  have modified `Array.prototype` we break can break their app. Using `import arrayIncludes from 'core-js/features/array/includes'` don't modify `Array.prototype` so it is safe to use

Reply:
Makes sense. ok, I'll change